Packs/CommonScripts/Scripts/IsInternalDomainName/IsInternalDomainName.yml

```diff
@@ -0,0 +1,43 @@
+args:
+- auto: PREDEFINED
+  default: true
+  description: A comma-separated list of main - domains.
+  isArray: true
+  name: domains_to_compare
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: true
+  description: A comma-separated list of subdomains.
+  isArray: true
+  name: domain_to_check
+  required: true
+  secret: false
+comment: Checks if a subdomain is located in at least one of the given domains to
+  compare with.
```

Comment:
@ShirleyDenkberg For example, is apps.paloaltonetworks.com is a part of paloaltonetworks.com
The script can accept multiple values for both arguments and will iterate through each of the domains we want to check if it is in the list of the domains we want to compare with. As long as the tested domain is in one of the compared domains the is internal will be set as true. For example our domains to check are apps.paloaltonetworks.com,and apps.paloaltonetworks.bla and the domains_to_compare are paloaltonetworks.com and demisto.com
The result for apps.paloaltonetworks.com will be true as its a part of paloaltonetworks.com and the result for apps.paloaltonetworks.bla will be false as its nether a part of paloaltonetworks.com or demisto.com